### PR TITLE
[incubator-kie-drools-5909] [new-parser] Accumulate parsed incorrectl…

### PR DESCRIPTION
**Pull request by tkobayas**
…y if init and action statements are empty

**Issue**: 

* https://github.com/apache/incubator-kie-drools/issues/5909

### CODE REVIEW

File: drools-drl/drools-drl-parser-tests/src/test/java/org/drools/drl/parser/antlr4/MiscDRLParserTest.java [hunk 1]
```diff
@@ -5122,4 +5122,25 @@ void javaKeywordsInPackage(String keyword) {
 
         assertThat(pkg.getRules().get(0).getName()).isEqualTo("R");
     }
+
+    @Test
+    void accumulateEmptyChunks() {
+        ParserTestUtils.enableOldParser();
```
Comment (yurloc) [inline drools-drl/drools-drl-parser-tests/src/test/java/org/drools/drl/parser/antlr4/MiscDRLParserTest.java]:
```suggestion
```

**Review by yurloc — APPROVED** [review]
One unintentional line, otherwise LGTM.

File: drools-drl/drools-drl-parser-tests/src/test/java/org/drools/drl/parser/antlr4/MiscDRLParserTest.java (hunk 1 above)
Comment (tkobayas) [reply drools-drl/drools-drl-parser-tests/src/test/java/org/drools/drl/parser/antlr4/MiscDRLParserTest.java]: Ahh, thank you, and sorry for the lack of attention...

**Comment by tkobayas** [comment]
@mariofusco @gitgabrio Please review, thanks!

**Review by mariofusco — APPROVED** [review]

**Pull request merged**